src/common/MovieTile/styled.js

```diff
@@ -119,30 +156,37 @@ font-weight: 400;
 line-height: 4px;
 color: ${({ theme }) => theme.colors.woodsmoke};
 
- @media (max-width: ${({ theme }) => theme.breakpoint.mobileMin}px) {
+@media (max-width: ${({ theme }) => theme.breakpoint.mobileVertical}px) {
```

Comment:
hmm, please check again if the names of the breakpoints match ;) because here you can see that you have changed the breakpoint to "mobileVertical", and in the theme.js file completely different names of these points, look here:

        tablet: 992,
        mobileMax: 767,

Reply:
thanks for noticing!

Reply:
you're welcome 👍